auto-docs

Suryansh5545 opened the pull request:
I have successfully created docs automatically from docstring using sphinx and i have used theme sphinx-rtd-theme and have edited it to look exactly like the previous docs html. The problem is the command  and there description  Which is in the middle is not exaclty same as previous one. and i dont know how can i control or edit it? 

THE Result:-
![Selection_059](https://user-images.githubusercontent.com/34577232/71547977-46bc4700-29cd-11ea-8ccf-6cbee568ec45.png)

Expected:-
![Selection_060](https://user-images.githubusercontent.com/34577232/71547989-6b182380-29cd-11ea-9b1e-2a29ecaad886.png)

Suryansh5545 commented:
Also the person creating or editing will require the edited theme so that it look similar to old one.

yashdusing commented:
@Suryansh5545 could you add the screenshot to the list of commands ?

yashdusing commented:
Also, add the procedure to create the docs in the PR and the readme if possible

yashdusing commented:
@Ram81 @pushkalkatara @RishabhJain2018 please review

Suryansh5545 commented:
This is how the command list look in the newly auto generator docs. It doesn't look satisfactory to me .

![Selection_062](https://user-images.githubusercontent.com/34577232/71548202-fb0b9c80-29d0-11ea-96c3-62c239762952.png)

yashdusing commented:
@Suryansh5545 Could you please work on the spacing between the html elements ?

Suryansh5545 commented:
Sir I have deleted old files which were not required now, have included the newly created evalai-theme , and created a README.md file. But i was unable to change the main style of how docs appear i can change the layout but can not influence them so i wasn't able to correct the spacing between html elements

pushkalkatara commented:
@Suryansh5545 Please try to improve the theme. Maybe you can work on to get the previous one.
![output](https://user-images.githubusercontent.com/21266230/71585789-cb2dd780-2b3d-11ea-9939-3d0e9a5cb6d0.png)

Maybe we can adopt a new theme, my fav one was the sphinx rtd theme - https://github.com/snide/sphinx_rtd_theme

What do you think @yashdusing?

Suryansh5545 commented:
Should I work on using sphinx rtx theme

Suryansh5545 commented:
Sir I have implemented the sphinx_rtd_theme and i also removed some modules from the evalai.rtd file as they have no doc string and were covering space

Suryansh5545 commented:
![Selection_007](https://user-images.githubusercontent.com/34577232/71613759-2b6c5a00-2bce-11ea-98a0-3e4b1026b809.png)
